www/app.py

```diff
@@ -36,5 +44,37 @@ def getblockdiagram(id):
         content = content_file.read()
     return Response(content, mimetype='text/xml')
 
+@app.route('/api/v1/sendcommand', methods = ['POST'])
+def sendCommand():
+    print request.form
+    runCommand(request.json)
+    return jsonify(request.json)
+
+def init_rover_service():
```

Comment:
I think we need to stay consistent with the style of function naming. Right now this file would have all lowercase, camelcase, and lowercase with underscores for function names. The Python [style guide](https://www.python.org/dev/peps/pep-0008/) says to use lowercase with underscores, so I would suggest we change to that. We didn't exactly do that for the other function names, but I think we need to decide something now and stick to it.